spyder/widgets/explorer.py

```diff
@@ -45,15 +45,15 @@
 except:
     nbexporter = None    # analysis:ignore
 
+
 def open_file_in_external_explorer(filename):
     if sys.platform == "darwin":
         subprocess.call(["open", "-R", filename])
+    if os.name == 'nt':

```

Comment:
I think this should be change it for `elif os.name == 'nt':`

```
if sys.platform == "darwin":
    ...
elif os.name == 'nt':
   ...
else:
    ...
```

if not in macOS it will execute both mac and linux command